src/elements/datepicker/datepicker-toggle/datepicker-toggle.component.ts

```diff
@@ -132,7 +132,7 @@ class SbbDatepickerToggleElement<T = Date>
       return;
     }
     this._calendarElement.wide = this.datepicker.wide;
-    this._calendarElement.now = this._nowOrNull();
+    this._calendarElement.now = this._nowOrNull()!;
```

Comment:
Looks strange to do this as it can be null. why not using T | null for dates?

Reply:
We do. It is just `now` that is different.
I also was not quite happy with this, but I'm not sure how much effort to invest in `now`.
Thoughts?

Reply:
I think I maybe understand it now, because the now property of the calendarElement has a fallback, we can assign null and it fallbacks to today(); but in theory now doesn't allow null.

Reply:
Ok, no. I think this property will be removed in the refactoring anyway, so I'll leave it as it is.